src/pre-commit

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+const childProcessExec = require('child_process').exec;
+const util = require('util');
+
+const exec = util.promisify(childProcessExec);
+
+if (!String.prototype.startsWith) {
+    Object.defineProperty(String.prototype, 'startsWith', {
+        value: function(search, rawPos) {
+            var pos = rawPos > 0 ? rawPos|0 : 0;
+            return this.substring(pos, pos + search.length) === search;
+        }
+    });
+}
+
+checkCommitMessage();
+
+async function checkCommitMessage(){

```

Comment:
Hmm, I am not really a fan of all of this logic being duplicated.

Can we create a third shared script between the two commands instead?
That will be much easier for us to maintain. 

And again, we can just use `ncc` to have single file structure afterwards

Reply:
Yes I am working on movind it to a shared file atm 👍

Reply:
Yeah sorry, I was too lazy to setup github actions to run `ncc` automatically on pushes to main 👎